src/sssom/writers.py

```diff
@@ -302,7 +339,7 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
         ],
         "version": metadata.get("mapping_set_version", ""),
         "name": name,
-        "title": name,
+        "title": name,  # TODO -> mapping_set_description?
```

Comment:
Or `mappings_set_title`

Reply:
Thanks I'll update my comments, and probably just pull from that field maybe instead. I don't know if that field was available when I started working on this but looks perfect

Reply:
Changed. SSSOM `mapping_set_title` now maps to ConceptMap `title`. CC FYI: @ShahimEssaid